src/main/io/displayport_max7456.c

```diff
@@ -152,6 +153,16 @@ static bool layerCopy(displayPort_t *displayPort, displayPortLayer_e destLayer,
     return max7456LayerCopy(destLayer, sourceLayer);
 }
 
+static int writeFontCharacter(displayPort_t *instance, uint16_t addr, const osdCharacter_t *chr)
```

Comment:
Why the `int` return value here and not bool?

Reply:
The was to let the drivers return the number of bytes used, but it turns out that's not that useful right now. I'll change it to bool.